src/receivers/ExpressReceiver.ts

```diff
@@ -97,6 +96,13 @@ export interface ExpressReceiverOptions {
   app?: Application;
   router?: IRouter;
   customPropertiesExtractor?: (request: Request) => StringIndexed;
+  dispatchErrorHandler?: (args: ReceiverDispatchErrorHandlerArgs) => Promise<void>;
+  processEventErrorHandler?: (args: ReceiverProcessEventErrorHandlerArgs) => Promise<boolean>;
+  // NOTE: for the compatibility with HTTPResponseAck, this handler is not async
+  // If we receive requests to provide async version of this handler,
+  // we can add a different name function for it.
+  unhandledRequestHandler?: (args: ReceiverUnhandledRequestHandlerArgs) => void;
+  unhandledRequestTimeoutMillis?: number;
```

Comment:
At what point do you think we should factor these handlers out to the top-level `Receiver` interface? I think it's too much to ask for that in this PR, but perhaps we can file an issue to look through all the receivers and factor out the common properties and methods into `Receiver`? WDYT?

Reply:
@filmaj thanks for the comment. 

These error handlers rely on Node.js http module. `AwsLambdaRecevier` and `SocketModeReceiver` are not compatible with the interface (As for `SocketModeReceiver`, only the OAuth flow parts rely on http module, though).

> unhandledRequestTimeoutMillis

I think that eventually this option can be a common one. However, at this moment, `AwsLambdaReceiver` and `SocketModeReceiver` do not support this option. When we consistently support this option across all the built-in receivers, we can consider adding this to `Receiver` interface (or the `App` parameters).

Reply:
Ah right, thank you for clarifying.